src/database/FirehoseConnector.js

```diff
@@ -206,6 +209,9 @@ class FirehoseConnector {
             meetingUrl,
             meetingUniqueId,
             endpointId,
+            conferenceStartTime,
+            sessionStartTime,
+            sessionEndTime,
             sessionDurationMs,
             conferenceDurationMs,
```

Comment:
If we expose now the start and end times, do we still need to calculate and expose the duration's?
Or could we just drop the code for the duration's as these will get calculated on the SQL anyway?

Reply:
Fair point, but one thing to consider is that the data team uses conferenceDuration in a bunch of their queries, and having to calculate it for each entry might be costly when doing the queries so maybe we keep it in order to have more efficient queries in redshift in the detriment of having the logic in rtcstats?